guides/plugins/plugins/administration/using-custom-fields.md

```diff
@@ -0,0 +1,100 @@
+# Using custom fields
+
+## Prerequisites
+
+All you need for this guide is a running Shopware 6 instance, the files and preferably a registered module in your 
+own plugin. Don't know how to create an own plugin yet? Head over to the following guide:
+
+{% page-ref page="./../plugin-base-guide.md" %}
+
+In order to craft your module, you will need to create lots on own components. If you're not sure about how to do that,
+take a look at the corresponding guide:
+
+{% page-ref page="./../administration/add-custom-component.md" %}
+
+In addition, of course you need an entity with custom fields to be able to add those custom fields to your module to
+begin with. Here you can learn how to add your custom fields:
+
+{% page-ref page="./../framework/custom-field/add-custom-field.md" %}
+
+## Using custom fields in your module
+
+In Shopware, we provide an own component called `sw-custom-field-set-renderer` for your template, being tailored 
+specifically to display custom field sets. 
+
+As a consequence, you're of course able to use this component to display your custom fields. See here:
+
+{% code title="<plugin-root>/src/Resources/app/administration/app/src/component/swag-basic-example/swag-basic-example.html.twig" %}
+```twig
```

Comment:
html should provide better highlighting I guess and there are no twig blocks in the example.